### PR TITLE
Code samples for TAP [2]

### PR DESCRIPTION
**Pull request by aaronforinton-dojo**

### CODE REVIEW

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+val baseAmount = bundle.getInt("com.dojo.extra.BASE_AMOUNT”, -1)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:1]:
```suggestion
val baseAmount = bundle.getInt("com.dojo.extra.BASE_AMOUNT", -1)
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+val gratuityAmount = bundle.getInt("com.dojo.extra.GRATUITY_AMOUNT”, -1)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:2]:
```suggestion
val gratuityAmount = bundle.getInt("com.dojo.extra.GRATUITY_AMOUNT", -1)
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+val txResult = bundle.getString("com.dojo.extra.TRANSACTION_RESULT”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:3]:
```suggestion
val txResult = bundle.getString("com.dojo.extra.TRANSACTION_RESULT")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+val authCode = bundle.getString("com.dojo.extra.AUTH_CODE”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:4]:
```suggestion
val authCode = bundle.getString("com.dojo.extra.AUTH_CODE")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+val txId = bundle.getString("com.dojo.extra.TRANSACTION_ID”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:5]:
```suggestion
val txId = bundle.getString("com.dojo.extra.TRANSACTION_ID")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+val paymentMethod = bundle.getString("com.dojo.extra.PAYMENT_METHOD”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:6]:
```suggestion
val paymentMethod = bundle.getString("com.dojo.extra.PAYMENT_METHOD")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+if (paymentMethod == "KEYED”) return
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:8]:
```suggestion
if (paymentMethod == "KEYED") return
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+val cvmString =  bundle.getString("com.dojo.extra.CARDHOLDER_VERIFICATION_METHOD”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:10]:
```suggestion
val cvmString =  bundle.getString("com.dojo.extra.CARDHOLDER_VERIFICATION_METHOD")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+val cardScheme = bundle.getString("com.dojo.extra.CARD_SCHEME”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:11]:
```suggestion
val cardScheme = bundle.getString("com.dojo.extra.CARD_SCHEME")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+val appLabel = bundle.getString("com.dojo.extra.CARD_APPLICATION_LABEL”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:12]:
```suggestion
val appLabel = bundle.getString("com.dojo.extra.CARD_APPLICATION_LABEL")
```

File: mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+val appId = bundle.getString("com.dojo.extra.CARD_APPLICATION_ID”)
```
Comment (monica-m-ps) [inline mobile-sdk/android/tap-to-pay/kotlin/bundle-item.kts:13]:
```suggestion
val appId = bundle.getString("com.dojo.extra.CARD_APPLICATION_ID")
```

**Review by monica-m-ps — CHANGES_REQUESTED** [review]
Just a tiny one regarding the " sign.

**Comment by aaronforinton-dojo** [comment]
Oh thank you, I had only caught some of those

**Review by monica-m-ps — APPROVED** [review]

**Pull request merged**